subprojects/tooling-api/src/crossVersionTest/groovy/org/gradle/integtests/tooling/r47/BuildProgressCrossVersionSpec.groovy

```diff
@@ -24,7 +24,8 @@ import org.gradle.test.fixtures.server.http.MavenHttpRepository
 import org.gradle.test.fixtures.server.http.RepositoryHttpServer
 import org.gradle.tooling.ProjectConnection
 
-@TargetGradleVersion(">=4.7")
+@TargetGradleVersion(">=4.7 <5.6")
+// As of 5.6, we no longer rely on the Maven Aether libraries for `maven-publish`
```

Comment:
It's not clear to me why this test is specific to the Aether libraries.  We still have to do the GET for the xml file and sha1, but we don't generate download progress events for those now?  Should we?

Reply:
Thanks for prompting me to look at this again. With the new implementation, we download the `maven-metadata.xml` file once per publish, and never download the `.sha1` file.

I've updated the test to verify the progress events for both the old and the new implementations: aada87c2f546efb05f79a622303fa1dee74826f0